Testing/functional/e2e/cypress/integration/timeline/comments.js

```diff
@@ -10,6 +12,25 @@ function verifyAdd() {
         .click();
     cy.get('[data-testid=commentText]')
         .contains('test comment goes here!');
+    
+    cy.get("@tokens").then(tokens => {
+        cy.request({
+            url: `${BASEURL}${HDID}/Comment/`,
+            followRedirect: false,
+            auth: {
+                bearer: tokens.access_token
+            },
+            headers: {
+                accept: 'application/json'
+            }
+        })
+        .should((response) => { 
+            expect(response.status).to.eq(200)
+            expect(response.body).to.not.be.null
+            cy.log(`response.body: ${JSON.stringify(response.body)}`)
+            expect(JSON.stringify(response.body.resourcePayload)).contains(`"entryTypeCode":"${entryTypeCode}"`);
```

Comment:
for info, here is the resourcePayload that we need to validate against. Note that we can't rely on the key "9529523" as it comes from the external service.
`{"resourcePayload":{"9529523":[{"id":"9f33abb2-6bb1-469f-a251-5717612e7e01","userProfileId":"P6FFO433A5WPMVTGM7T4ZVWBKCSVNAYGTWTU3J2LWMGUMERKI72A","text":"test comment goes here!","entryTypeCode":"Med","parentEntryId":"9529523"`